components/cop_codegen/src/rpn_function.rs

```diff
@@ -255,6 +256,24 @@ fn parse_arg_type(arg: &FnArg) -> Result<TypePath> {
     Ok(eval_type.clone())
 }
 
+fn check_attr(meta: &[Ident], item_fn: &ItemFn) -> Result<()> {
+    if item_fn.decl.inputs.len() < meta.len() {
+        return Err(meta[0]
```

Comment:
Can we use `proc_macro::Span::call_site().error` instead of `meta[0].span().unwrap()`?

Reply:
Can `Span::call_site()` point to the position in the input?

Reply:
@H-ZeX Then, here I suggest use the span of `meta[item_fn.decl.inputs.len()]` (of course save the length to a variable first)
And below, use the meta of `m` instead of `meta[0]`

Reply:
@lonng The the doc of `proc_macro2::Span`
``` 
Panics
Panics if called from outside of a procedural macro. Unlike proc_macro2::Span, the proc_macro::Span type can only exist within the context of a procedural macro invocation.` 
```
So this panic harmless.